src/modules/Builder/features/ActivityAbout/ActivityAbout.tsx

```diff
@@ -31,9 +37,33 @@ export const ActivityAbout = () => {
   ]);
 
   const { control, setValue, watch } = useFormContext();
-
   const { name } = useCurrentActivity();
 
+  const activities = watch('activities');
+  const activityItems = watch(`${name}.items`);
+  const activityWithReviewable = activities?.find((activity: ActivityFormValues, index: number) => {
+    const activityName = `activities.${index}`;
+
+    if (name === activityName) return false;
+
+    return activity.isReviewable;
+  });
+  const hasUnsupportedReviewableItemTypes = activityItems?.some(
+    (item: ItemFormValues) =>
+      ![
```

Comment:
looks like this array list can be moved into a constant:
const availableItems = [
        ItemResponseType.SingleSelection,
        ItemResponseType.MultipleSelection,
        ItemResponseType.Slider
]
```suggestion
      ![...availableItems,
```

Reply:
done